### ISSUE TITLE
Title: Incorrect Return Value in `is_dismissed` Method

### ISSUE THREAD

**Issue by heflavie** [LupaDevStudio/Geozzle#25]
#### Description:

In the file `tools/kivads.py`, there is a mistake in the way the method `setFullScreenContentCallback` is used when showing interstitial ads and rewarded ads. Instead of directly calling `setFullScreenContentCallback` on the interstitial ad or rewarded ad object, the code assigns a value to it, which is incorrect.
The current implementation is:

```python
def is_dismissed(self):
    if self.full_screen_callback.dismissed:
        return True
    else:
        False
```

In Python, when a return statement is omitted, the function implicitly returns `None`. This can lead to unexpected behavior when checking the return value of `is_dismissed()`, as it will not return `False` as expected in some cases.

#### Expected Behavior:
The function should always return either `True` or `False`. To fix this, the `else` block should explicitly return `False`:

```python
def is_dismissed(self):
    if self.full_screen_callback.dismissed:
        return True
    else:
        return False
```

This change ensures that the function always returns a boolean value, avoiding potential issues caused by returning `None`.

#### Steps to Reproduce:

1. Call `is_dismissed()` on an instance where `self.full_screen_callback.dismissed` is `False`.
2. Observe that the function returns `None` instead of `False`.

**Comment by LupaDevStudio** [comment]
I don't think this will change anything to the behavior of the application but that's right, thanks for noticing.
